.script/utils/gitWrapper.ts

```diff
@@ -0,0 +1,33 @@
+import { cli, devOps } from '@azure/avocado';
+import * as logger from './logger';
+import './stringExtenssions'
+
+export async function GetDiffFiles(fileTypeSuffixes?: string[], filePathFolderPreffixes?: string[]){
+    const config = cli.defaultConfig();
+    const pr = await devOps.createPullRequestProperties(config);
+  
+    if (pr === undefined) {
```

Comment:
That not the way to check for undefined in JavaScript
```suggestion
    if (typeof pr === "undefined") {
```

Reply:
Thanks!!